### PR TITLE
Initial manual test UI

### LINKED ISSUE #22: Find a place to manually trigger "Wifi Periodic Scan" test.
Find a way to run the "Wifi Periodic Scan" test from testrtc. Due to its nature (multiple minutes) it cannot be run by default and one may need to have a UI to allow to trigger it.

For now the test is/will be implemented under the "?test_filter=Wifi Periodic Scan" flag.

### PR DESCRIPTION
**Pull request by KaptenJansson**
Fixes #22

### CODE REVIEW

File: src/js/main.js [hunk 1]
```diff
@@ -124,7 +127,7 @@ function Test(suite, name, func) {
 
   var toolbar = document.createElement('core-toolbar');
   toolbar.setAttribute('class', 'test');
-  var title = document.createElement('span');
+  var title = document.createElement('div');
```
Comment (KaptenJansson) [inline src/js/main.js]: This results in a larger progress bar and also gives the new manual start buttons some more room.

**Comment by andresusanopinto** [comment]
This breaks in quite some scenarios....

E.g. since we use global functions for tests, if a manual test is running and the automatic tests are started this just totally misbehaves.


But maybe you just put up this CL to start the discussiong on the UI?

**Comment by KaptenJansson** [comment]
I wanted to reuse as much as possible of the existing code but I do realize the limitations of the global functions. But your exact scenario can be solved by just disabling the start button for the auto tests, I mean it does not make sense to be able to run those at the same time.

Having said that, maybe it makes sense to separate the manual tests entirely from the auto test flow? I repeat what I wrote in the beginning, I wanted to reuse as much as possible.

And yes you are correct, I wanted to create a "demo" of the manual tests that mostly work so we can discuss how this should be handled.

**Comment by KaptenJansson** [comment]
I will close this and create a new one once I'm done.

**Pull request closed without merging**